include/ppx/grfx/grfx_command.h

```diff
@@ -300,13 +300,31 @@ class CommandBuffer
         uint32_t                          setCount,
         const grfx::DescriptorSet* const* ppSets) = 0;
 
+    //
+    // Parameters count and offset are measured in DWORDs (uint32_t) aka 32-bit values.
+    // To set the first 4 32-bit values, use count = 4, offset = 0.
+    // To set the 16 DWORDs starting at offset 8, use count=16, offset = 8.
+    //
+    virtual void SetGraphicsPushConstants(
+        const grfx::PipelineInterface* pInterface,
+        uint32_t                       count,
+        const void*                    pValues,
```

Comment:
Why a `void*`? the comment above says it's handled as 32-bit words, so why not `uint32_t*`? This way the type tells us how to handle count, as with a void, it's a bit unclear (from prototype, comment of course makes it clear).

Reply:
Yeah, right, so uint32_t isn't quite right either. Fine for void* then!